systems/plants/RigidBodyManipulator.h

```diff
@@ -62,26 +62,34 @@ class DLLEXPORT_RBM RigidBodyLoop
 
 public:
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+	friend std::ostream& operator<<(std::ostream& os, const RigidBodyLoop& obj);
 };
 
 class DLLEXPORT_RBM RigidBodyManipulator
 {
 public:
   RigidBodyManipulator(int num_dof, int num_featherstone_bodies=-1, int num_rigid_body_objects=-1, int num_rigid_body_frames=0);
-  RigidBodyManipulator(const std::string &urdf_filename);
+  RigidBodyManipulator(const std::string &urdf_filename, const std::string &floating_base_type = "rpy");
```

Comment:
I think this should be an enum instead of a string. This makes it really clear what the options are, you don't have to check for illegal inputs, and it's faster (that last one is a very minor point). In general, we should really start using enums more.